test/unittests/execute_test.cpp

```diff
@@ -1003,9 +1004,10 @@ TEST(execute, reuse_args)
         "0061736d01000000010b0260027e7e017e6000017e03030200010a19020e002000200180210120002001820b08"
         "004217420510000b");
 
-    const std::vector<uint64_t> args{20, 3};
+    const std::array<uint64_t, 2> args{20, 3};
```

Comment:
Why is this change needed?

Reply:
Not needed, but no need to use `std::vector` to allocate 2 args.